api/src/tests/resource_groups.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use super::new_test_context;
+use aptos_api_test_context::current_function_name;
+use aptos_cached_packages::aptos_stdlib;
+use aptos_framework::BuiltPackage;
+use std::path::PathBuf;
+
+// This test verifies that both READ APIs can seamlessly translate from resource group to resource
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+async fn test_read_resoure_group() {
```

Comment:
What do the failure cases look like (i.e. what errors are returned) and do we need test cases for these:
1. If the client is asking for a member resource in a resource group that doesn't exist.
2. If the client is asking for a member resource that doesn't exist in a resource group that exists.